tools/xwiishow.c

```diff
@@ -2169,7 +2169,7 @@ static void led_refresh(int n)
 
 	ret = xwii_iface_get_led(iface, XWII_LED(n+1), &led_state[n]);
 	if (ret)
-		print_error("Error: Cannot read LED state");
+		print_error("Error: Cannot read LED #%i state",n+1);
```

Comment:
Unrelated change

Reply:
Dropped in #111.  Will create a separate pull request for this one.